dc-cudami-editor/src/components/modals/ImageAdderModal.jsx

```diff
@@ -87,11 +82,10 @@ class ImageAdderModal extends Component {
   }
 
   addImageToEditor = (resourceId) => {
-    const filteredAttributes = Object.fromEntries(
-      Object.entries(this.state.attributes).filter(([_, value]) => value !== '')
-    )
     const data = {
-      ...filteredAttributes,
+      ...mapValues(this.state.attributes, (value) =>
```

Comment:
Wie funktioniert das hier? `mapValues` gibt ja einen Array zurück, aber du destructurst den in ein Objekt?